thirdparty/openssl/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ endif()
 set(CFG_OPTS "${CFG_OPTS} no-asm no-idea no-mdc2 no-rc5 no-ssl2 no-ssl3")
 set(CFG_CMD sh -c "${CFG_ENV_VAR} ${SOURCE_DIR}/${CONFIG_SCRIPT} ${CFG_OPTS}")
 
-set(BUILD_CMD "$(MAKE) depend && $(MAKE) CC=\"${CC}\" SHARED_LDFLAGS=\"${SHARED_LDFLAGS}\" LD=\"${LD}\" RANLIB=\"${RANLIB}\"")
+set(BUILD_CMD "$(MAKE) CC=\"${CC}\" SHARED_LDFLAGS=\"${SHARED_LDFLAGS}\" LD=\"${LD}\" RANLIB=\"${RANLIB}\"")
```

Comment:
It's probably okay, although this message doesn't inspire much confidence, but what does it do on Mac OS other than save some time?

```
*** Because of configuration changes, you MUST do the following before
*** building:

        make depend
```

Reply:
The following `depend` target will do that. Or does the first `$(MAKE) depend` do other tricks?

` set(BUILD_CMD sh -c "${BUILD_CMD} --silent depend build_crypto build_ssl >/dev/null 2>&1")`

Reply:
My bad. It was just written the opposite way from how I would intuitively write it (targets first, then vars) so I stupidly overlooked it.